### PR TITLE
[MM-54850] Use request-specific logger in channels/app/admin_advisor.go

### PR DESCRIPTION
**Pull request by hanzei**
#### Summary
This is an example PR for the community campaign that tackles https://mattermost.atlassian.net/browse/MM-45066.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-54850

#### Release Note
```release-note
NONE
```

### CODE REVIEW

File: server/channels/app/admin_advisor.go [hunk 1]
```diff
@@ -14,7 +14,7 @@ import (
 	"github.com/mattermost/mattermost/server/v8/platform/shared/mail"
 )
 
-func (a *App) GetWarnMetricsStatus() (map[string]*model.WarnMetricStatus, *model.AppError) {
+func (a *App) GetWarnMetricsStatus(rctx request.CTX) (map[string]*model.WarnMetricStatus, *model.AppError) {
```
Comment (hanzei) [inline server/channels/app/admin_advisor.go:17]: Naming per https://community.mattermost.com/core/pl/f7oo4n4aijn89mdz4f65uak65w

**Comment by hanzei** [comment]
cc @jespino

**Review by fmartingr — APPROVED** [review]
👍 Thanks for taking care of this.

**Review by agnivade — APPROVED** [review]
Great work!

**Pull request merged**